src/domain/journey/subspace/layout/SubspacePageLayout.tsx

```
@@ -321,3 +304,37 @@ const SubspacePageLayout = ({
 };
 
 export default SubspacePageLayout;
+
+export const SubspaceInnovationFlow = createLayout(({ children }: PropsWithChildren<{}>) => {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)